.github/workflows/node.yml

```diff
@@ -2,7 +2,7 @@ name: Node.js CI
 
 on:
   pull_request:
-    paths: ['ui/**']
+    paths: ['ui/**', '**.go']
```

Comment:
The title of the PR says always run for backend changes, but this says just run very specifically for .go files.  What's the goal?

Reply:
Our backend changes are triggered by **.go.  So this is the same.  If this is missing important stuff, so is our backend trigger.  Do you have an instance where you think this is wrong?

Reply:
Changes to go.mod?

Reply:
Or maybe the Makefile that builds the go stuff could have a subtle change in the way things are built?

Reply:
The action ends up calling 

```run: yarn chain:build && yarn build```

So it builds and uses just about everything.  We can't limit the file set without a lot of work and maintenance.

Reply:
I removed the paths dependencies so we always get this.